src/applications/post-911-gib-status/tests/post-911-gib-status-helpers.js

```diff
@@ -99,6 +99,16 @@ function initApplicationMock(token) {
     verb: 'get',
     value: backendStatus,
   });
+
+  mock(token, {
+    path: '/v0/feature_toggles',
+    verb: 'get',
+    value: {
+      data: {
+        features: [],
+      },
+    },
+  });
```

Comment:
I really love seeing feature toggles used like this. It sets up the E2E test really nicely for a smooth migration.